packages/node_modules/@webex/plugin-meetings/test/unit/spec/journey/testUtils.js

```diff
@@ -104,30 +104,40 @@ const waitForEvents = (scopeEvents) => {
     new Promise((resolve, reject) => {
       timer = setTimeout(() => {
         reject(new Error(' Events not triggred'));
-      }, 40000);
+      }, 29000);
     }),
     new Promise((resolve, reject) => {
       try {
         const result = [];
 
-        console.log('CHECKING EVENTS ', scopeEvents);
-
         scopeEvents.forEach((obj) => {
           obj.scope.once(obj.event, (value) => {
             result.push(Object.assign(obj, {result: value}));
             if (obj.event === 'meeting:added') {
               obj.user.meeting = value.meeting;
+              obj.user.meeting.name = obj.user.name;
+              obj.user.memberId = value.meeting.locusInfo.parsedLocus.selfId;
+            }
+            if (obj.event === 'meeting:removed') {
+              console.log('MEETING:REMOVED BEFORE', value);
```

Comment:
Use `debug()` instead so console output can be controlled by the developer.

Reply:
matt whats the syntax ?

Reply:
@arun3528 [example](https://github.com/mattnorris/spark-js-sdk/blob/b4214c71a359bc476abc6f37f6c11a47da4df71b/packages/node_modules/%40ciscospark/plugin-messages/src/messages.js#L17)

It allows you to specify what you want printed when you run the tests. In the example, I used 'messages'. In this case you may want to do 'meetings:journey'. You would then do this to debug: 

```sh
DEBUG="meetings:journey" npm test -- --package @webex/plugin-meetings
```